R/metab.bookkeep.R

```diff
@@ -64,7 +64,7 @@ metab.bookkeep <- function(do.obs, do.sat, k.gas, z.mix, irr, ...){
   
 	if("datetime"%in%names(mb.args)){ # check to see if datetime is in the ... args
 		datetime <- mb.args$datetime # extract datetime
-		freq <- calc.freq(datetime) # calculate sampling frequency from datetime
+		freq <- calc.freq(datetime) - 1 # calculate sampling frequency from datetime
```

Comment:
This all depends on your perspective of how many observations are supplied for a given frequency. If you have hourly data, do you have 25 or 24 observations? The *frequency* would of course be 24, but the number of observations could be 24 or 25.